src/views/FetchFlow.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="fetch-contents">
+    <div class="d-flex h-100 text-center">
+      <div class="cover-container d-flex w-100 h-100 p-3 mx-auto flex-column">
+        <main class="px-3">
+          <h2>{{message | trans}}</h2>
+          <router-link v-if="showNewButton" :to="route('flows.newFlow')"
+            title="trans('flow-builder.create-a-new-flow')"
+            class="mt-3 btn btn-outline-secondary mr-2 active">{{'flow-builder.new-flow' | trans}}</router-link>
+          <router-link v-if="flowLink" :to="flowLink"
+              title="trans('flow-builder.edit-flow')"
+              class="mt-3 btn btn-outline-secondary mr-2 active">{{'flow-builder.go-to-flow' | trans}}</router-link>
+        </main>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+
+import { forEach, isEmpty } from 'lodash'
+import lang from '@/lib/filters/lang'
+import Routes from '@/lib/mixins/Routes'
+import { Component, Prop } from 'vue-property-decorator'
+import Vue from 'vue'
+import {Getter, Mutation, namespace} from 'vuex-class'
+import {store} from '@/store'
+const flowVuexNamespace = namespace('flow')
+import {IFlow} from '@floip/flow-runner'
+
+@Component<any>(
```

Comment:
@jcbashdown , can you try removing this `<any>` for new added components, I think it won't be necessary. (For existing component, I removed those `<any>` on vmo-2623 PR https://github.com/FLOIP/flow-builder/pull/55/files)

Reply:
CC/ @poojahpatil90